[BUG] Падает код экшн провайдер в случае если сработавшая диагностика в VsCode не принадлежит Bslls

**Версия**
0.17.1

**Описание ошибки диагностики**
Падает код экшн провайдер в случае если сработавшая диагностика в VsCode не принадлежит Bslls

**Пример кода**
```bsl
	Результат = РегулярныеВыраженияСервер.Заменить(
		СтрокаПоиска,
		РегулярноеВыражение,
		СтрокаЗамены
	);
```

**Скриншоты**
![GIF 12 01 2021 11-48-09](https://user-images.githubusercontent.com/32082417/104258979-3ab60100-54cc-11eb-88af-b6280b64ddd5.gif)

**Дополнительная информация**
```
[Error - 11:37:25] Request textDocument/codeAction failed.
  Message: Internal error.
  Code: -32603 
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "org.eclipse.lsp4j.jsonrpc.messages.Either.isLeft()" because "diagnosticCode" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Cannot invoke "org.eclipse.lsp4j.jsonrpc.messages.Either.isLeft()" because "diagnosticCode" is null
	at com.github._1c_syntax.bsl.languageserver.diagnostics.metadata.DiagnosticCode.getStringValue(DiagnosticCode.java:41)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at com.github._1c_syntax.bsl.languageserver.codeactions.DisableDiagnosticTriggeringSupplier.actionDisableDiagnostic(DisableDiagnosticTriggeringSupplier.java:171)
	at com.github._1c_syntax.bsl.languageserver.codeactions.DisableDiagnosticTriggeringSupplier.getDisableActionForLine(DisableDiagnosticTriggeringSupplier.java:122)
	at com.github._1c_syntax.bsl.languageserver.codeactions.DisableDiagnosticTriggeringSupplier.lambda$getCodeActions$1(DisableDiagnosticTriggeringSupplier.java:87)
	at java.base/java.util.Optional.ifPresent(Unknown Source)
	at com.github._1c_syntax.bsl.languageserver.codeactions.DisableDiagnosticTriggeringSupplier.getCodeActions(DisableDiagnosticTriggeringSupplier.java:85)
	at com.github._1c_syntax.bsl.languageserver.providers.CodeActionProvider.lambda$getCodeActions$0(CodeActionProvider.java:93)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(Unknown Source)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at com.github._1c_syntax.bsl.languageserver.providers.CodeActionProvider.getCodeActions(CodeActionProvider.java:96)
	at com.github._1c_syntax.bsl.languageserver.BSLTextDocumentService.lambda$codeAction$1(BSLTextDocumentService.java:164)
	... 7 more
```

--- a/src/main/java/com/github/_1c_syntax/bsl/languageserver/codeactions/DisableDiagnosticTriggeringSupplier.java
+++ b/src/main/java/com/github/_1c_syntax/bsl/languageserver/codeactions/DisableDiagnosticTriggeringSupplier.java
@@ -45,6 +45,8 @@ import java.util.Map;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static com.github._1c_syntax.bsl.languageserver.providers.DiagnosticProvider.SOURCE;
+
 
 @Component
 public class DisableDiagnosticTriggeringSupplier implements CodeActionSupplier {
@@ -160,10 +162,11 @@ public class DisableDiagnosticTriggeringSupplier implements CodeActionSupplier {
     return result;
   }
 
-  private List<CodeAction> actionDisableDiagnostic(Function <String, CodeAction> func, CodeActionParams params) {
+  private List<CodeAction> actionDisableDiagnostic(Function<String, CodeAction> func, CodeActionParams params) {
     return params.getContext()
       .getDiagnostics()
       .stream()
+      .filter(diagnostic -> SOURCE.equals(diagnostic.getSource()))
       .map(Diagnostic::getCode)
       .map(DiagnosticCode::getStringValue)
       .distinct()

--- a/src/test/java/com/github/_1c_syntax/bsl/languageserver/codeactions/DisableDiagnosticTriggeringSupplierTest.java
+++ b/src/test/java/com/github/_1c_syntax/bsl/languageserver/codeactions/DisableDiagnosticTriggeringSupplierTest.java
@@ -31,11 +31,11 @@ import org.eclipse.lsp4j.CodeAction;
 import org.eclipse.lsp4j.CodeActionContext;
 import org.eclipse.lsp4j.CodeActionParams;
 import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.Diagnostic;
 import org.eclipse.lsp4j.Range;
 import org.eclipse.lsp4j.TextDocumentIdentifier;
 import org.eclipse.lsp4j.jsonrpc.messages.Either;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
 
@@ -190,6 +190,32 @@ class DisableDiagnosticTriggeringSupplierTest {
 
     CodeActionContext codeActionContext = new CodeActionContext();
     codeActionContext.setDiagnostics(documentContext.getDiagnostics());
+
+    CodeActionParams params = new CodeActionParams();
+    params.setRange(new Range());
+    params.setTextDocument(textDocumentIdentifier);
+    params.setContext(codeActionContext);
+
+    List<Either<Command, CodeAction>> codeActions = codeActionProvider.getCodeActions(params, documentContext);
+
+    assertThat(codeActions)
+      .hasSize(1)
+      .extracting(Either::getRight)
+      .anyMatch(codeAction -> codeAction.getTitle().equals("Disable all diagnostic in file"));
+
+  }
+
+  @Test
+  void testNoBslLsDiagnostic() {
+
+    DocumentContext documentContext = TestUtils.getDocumentContextFromFile(
+      "./src/test/resources/suppliers/disableDiagnosticTriggeringEmpty.bsl"
+    );
+
+    TextDocumentIdentifier textDocumentIdentifier = new TextDocumentIdentifier(documentContext.getUri().toString());
+
+    CodeActionContext codeActionContext = new CodeActionContext();
+    codeActionContext.setDiagnostics(List.of(new Diagnostic()));
 
     CodeActionParams params = new CodeActionParams();
     params.setRange(new Range());

